core-web/libs/block-editor/src/lib/components/dot-block-editor/dot-block-editor.component.ts

```diff
@@ -81,6 +81,7 @@ export class DotBlockEditorComponent implements OnInit, OnDestroy {
             value
         );
     }
+    @Input() isFullscreen = false;
```

Comment:
I don't like this because I don't think the size of the block editor should be set by himself... but we can fix this later.